src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -419,6 +723,33 @@ void m()
         }
 
         [Fact()]
+        public void AnonymousTypeFieldCreatedInLambda_OperationTree()
+        {
+            string source = @"
+using System;
+class ClassA
+{
+    void m()
+    {
+        var o = (Action)(() => (/*<bind>*/new { x = 1, y = new { } }/*</bind>*/).ToString()); ;
+    }
+}
+";
```

Comment:
Nothing really, I was just creation cloned IOperation tests for existing tests in this source files.

---
In reply to: [123879628](https://github.com/dotnet/roslyn/pull/20364#discussion_r123879628) [](ancestors = 123879628)